src/vs/workbench/contrib/positronPreview/browser/previewHtml.ts

```diff
@@ -30,7 +30,7 @@ export class PreviewHtml extends PreviewWebview {
 		previewId: string,
 		webview: PreviewOverlayWebview,
 		readonly uri: URI,
-		readonly html: ShowHtmlFileEvent
+		readonly html?: ShowHtmlFileEvent
```

Comment:
this was edited to allow creation of `PreviewUrl` independent of a `ShowHtmlFileEvent`